contracts/mechs/token/BalanceTrackerFixedPriceToken.sol

```diff
@@ -68,16 +68,8 @@ contract BalanceTrackerFixedPriceToken is BalanceTrackerFixedPriceBase {
     /// @param amount Token amount.
     /// @return Received amount.
     function _getRequiredFunds(address requester, uint256 amount) internal virtual override returns (uint256) {
-        uint256 balanceBefore = IToken(olas).balanceOf(address(this));
```

Comment:
We trust OLAS contract and thus there's no need. The contract is specific to OLAS.

Reply:
Would add this as comment in the code?